core/math/color.h

```diff
@@ -241,6 +241,19 @@ struct Color {
 		b = p_c.b;
 		a = p_a;
 	}
+
+	Color(const String &p_code) {
+		if (html_is_valid(p_code)) {
+			*this = html(p_code);
+		} else {
+			*this = named(p_code);
+		}
+	}
+
+	Color(const String &p_code, float p_a) {
+		*this = Color(p_code);
+		a = p_a;
```

Comment:
```suggestion
		a *= p_a;
```

`Color("#ff000080", 1.0)` should be `#ff000080` or `#ff0000ff`?